docs/CHANGELOG/changelog-0.9.x.md

```
@@ -317,3 +317,14 @@ kubernetes:
 - Increased concurrency limits to decrease the build time of larger clusters. This change also affects Claudie's memory requirements, which should fit within 8 GB. [#1819](https://github.com/berops/claudie/pull/1819)
 - For autoscaled events, Terraformer will now skip refreshing the LoadBalancers and DNS infrastructure, if present. [#1830](https://github.com/berops/claudie/pull/1830)
+
+## v0.9.13
+
+## What's Changed
+- Concurrency limits are now configurable [#1838](https://github.com/berops/claudie/pull/1838)
+- Autoscaled nodepools are now limited to 256 nodes [#1839](https://github.com/berops/claudie/pull/1839)
+- Metadata secret will now be updated after node deletion [#1841](https://github.com/berops/claudie/pull/1841)
+- Builder TTL has been increased to 42 hours, as 2 hours is a small amount of time to react to issues [#1841](https://github.com/berops/claudie/pull/1850)
+
+## Bug fixes
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)